### PR TITLE
Makes set context functionality more reliable

### LINKED ISSUE #3124: Fix Set Context functionality for WebUI
It seems that once we selected necessary organization and location for our specific test and do verification that they are really selected - corresponding dropdown is not closed. That can affect other tests as some elements can be overlapped by that dropdown. As selenium replicates real user interactions - we can choose menu item instead clicking on expected element and that will give an exception for sure

### PR DESCRIPTION
**Pull request by oshtaier**
Closes #3124

Run few random tests to make sure that we do not brake anything:
```
nosetests tests/foreman/ui/test_location.py -m test_remove_environment
.
----------------------------------------------------------------------
Ran 1 test in 79.179s

OK
```
```
nosetests tests/foreman/ui/test_contentviews.py -m test_cv_create
..
----------------------------------------------------------------------
Ran 2 tests in 287.527s

OK
```
```
nosetests tests/foreman/ui/test_products.py -m test_positive_create_basic
.
----------------------------------------------------------------------
Ran 1 test in 123.422s

OK
```

### CODE REVIEW

File: robottelo/ui/base.py [hunk 1]
```diff
@@ -596,6 +595,16 @@ def perform_action_chain_move(self, locator):
             )
         ActionChains(self.browser).move_to_element(element).perform()
```
Comment (elyezer) [inline robottelo/ui/base.py]: wait_for_ajax is not needed here?

**Comment by elyezer** [comment]
ACK, pending question.

File: robottelo/ui/base.py (hunk 1 above)
Comment (oshtaier) [inline robottelo/ui/base.py]: I don't mind to add it, but no failures for that part for long-long time, so not sure if we need to add redundant waiter

Comment (elyezer) [inline robottelo/ui/base.py]: It is not needed, just trying to get more information. Thanks for the clarification.

Comment (oshtaier) [inline robottelo/ui/base.py]: Thanks for asking

**Comment by kbidarkar** [comment]
ACK.

**Pull request merged**